ext/pdo_pgsql/tests/common.phpt

```diff
@@ -18,7 +18,7 @@ if (false !== getenv('PDO_PGSQL_TEST_DSN')) {
 		$config['ENV']['PDOTEST_ATTR'] = getenv('PDO_PGSQL_TEST_ATTR');
 	}
 } else {
-	$config['ENV']['PDOTEST_DSN'] = 'pgsql:host=localhost port=5432 dbname=test user= password=';
+	$config['ENV']['PDOTEST_DSN'] = 'pgsql:host=localhost port=5432 dbname=test user=postgres password=postgres';
```

Comment:
@phackwer you can use PDO_PGSQL_TEST_DSN to set the connection string, so you can revert this piece just withith the same PR.

Thanks.